### DISCUSSION TITLE
Release binary's database

### DISCUSSION THREAD

**Discussion by ThePixelGamer** [naim94a/lumen#33] [Q&A] [answered]
Are the release binaries supposed to have a database with them or is that something that needs to be done separately?

**Comment by naim94a** [comment, marked as answer]
Releases currently require the user to setup a [PostgreSQL](https://www.postgresql.org/) instance separately. 

Using the [docker method](https://github.com/naim94a/lumen#docker-method) will setup a PostgreSQL instance automatically.

**Reply by ThePixelGamer** [reply]
Ah I see, in that case I'll go ahead and switch to the docker method.
